docs/source/guides/integrations.mdx

```diff
@@ -0,0 +1,170 @@
+# Integrate any ML framework with the Hub
+
+The Huggingface Hub makes it easy to host and share models with the community. It already has support for
+[dozens of libraries](https://huggingface.co/docs/hub/models-libraries) in the Open Source ecosystem. We are always
+working on expanding this support to push Machine Learning forward. The `huggingface_hub` library plays a key role in
+this process, allowing any Python script to easily push and load files.
+
+When talking about integrating a library to the Hub, we have 4 main topics:
+1. **Push to Hub:** implement a `push_to_hub()`-like method to upload a model to the Hub.
```

Comment:
I feel like directly mentioning `push_to_hub` and `from_pretrained` in this point of the guide introduces question marks to user, I'd rather just say what it does here but it's my opinion :D @stevhliu @Wauplin

Reply:
That's a great point! It does kind of assume you're familiar with the `push_to_hub` method. You could either describe it as Merve suggested, or add a link to the `push_to_hub` docstring :)

Reply:
there's already explanations on them as you go down in the guide, it's just I felt people might be intimidated not knowing what it is at the beginning written like this :)